src/app/shared/services/iam.service.ts

```diff
@@ -267,6 +270,22 @@ export class IamService {
     return chainConfig;
   }
 
+  private configureIpfsConfig() {
```

Comment:
I can write tests for this, if this is how we want it in this file.

Reply:
I think tests are not necessary here. It's just config file.